<ui version="4.0" >
 <class>WidgetExtensionPolicy</class>
 <widget class="QWidget" name="WidgetExtensionPolicy" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBoxFilterPolicy" >
     <property name="title" >
      <string>Choose what part of files you want to rename:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QRadioButton" name="radioButtonFilterBasename" >
        <property name="text" >
         <string>The basename (without extension)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="radioButtonFilterAll" >
        <property name="text" >
         <string>The entire filename (with extension)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QRadioButton" name="radioButtonFilterExtension" >
        <property name="text" >
         <string>Only the extension</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QPushButton" name="pushButtonReset" >
     <property name="text" >
      <string>Reset to default settings</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBoxExtensionDefinition" >
     <property name="title" >
      <string>The file extension starts just after the:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="1" column="0" colspan="5" >
       <widget class="QRadioButton" name="radioButtonExtensionDefinitionLastPoint" >
        <property name="text" >
         <string>First point from the right</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="5" >
       <widget class="QRadioButton" name="radioButtonExtensionDefinitionFirstPoint" >
        <property name="text" >
         <string>First point from the left</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QRadioButton" name="radioButtonExtensionDefinitionNthPoint" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QLabel" name="labelExtensionNthPoint" >
        <property name="text" >
         <string>th point from the right</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QSpinBox" name="spinBoxExtensionNthPoint" >
        <property name="minimum" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="4" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>42</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
